### PR TITLE
Migrate highlight file refernces to use @malloydata/syntax-highlight

### PR DESCRIPTION
**Pull request by ksittner**

### CODE REVIEW

**Review by whscullin — COMMENTED** [review]
You should delete the old files, and make sure to add an exception to .vscodeignore for the module like here: https://github.com/malloydata/malloy-vscode-extension/blob/main/.vscodeignore#L5

File: tsconfig.json [hunk 1]
```diff
@@ -13,6 +13,6 @@
     "noImplicitAny": true,
     "strict": true
   },
-  "include": ["./src"],
+  "include": ["./src", "./node_modules/@malloydata/syntax-highlight/grammars"],
```
Comment (whscullin) [inline tsconfig.json:16]: This probably isn't necessary.

File: package.json [hunk 2]
```diff
@@ -588,6 +589,7 @@
   },
   "devDependencies": {
     "@jest/globals": "^26.6.2",
+    "@malloydata/syntax-highlight": "^0.0.82",
```
Comment (whscullin) [inline package.json:592]: This should be in "dependencies" with the other malloy packages, and we update all malloy package versions in sync.

Comment (ksittner) [reply package.json:592]: Completed

Comment (ksittner) [reply package.json]: This and the undeleted files were an artifact of untracked changes. That and versioning have been resolved

**Comment by mtoy-googly-moogly** [comment]
Closing because @ksittner is no longer a legal contributor according to the machine. All code is now in #250 and I responded to the one change request there also.

**Pull request closed without merging**